sample-queries/sample-queries.json

```diff
@@ -3159,6 +3159,60 @@
       "postBody": "{\r\n    \"updateCategory\": \"feature\",\r\n    \"assets\": [\r\n        {\r\n            \"@odata.type\": \"#microsoft.graph.windowsUpdates.azureADDevice\",\r\n            \"id\": \"{azureAdDeviceId}\"\r\n        }\r\n    ]\r\n}",
       "tip": "Please enable the WindowsUpdates.ReadWrite.All permission to use this query.",
       "skipTest": false
+    },
+    {
+      "id": "814ffcd1-d2e2-460c-a64a-1253657c4978",
+      "category": "Edge",
+      "method": "GET",
+      "humanName": "list IE Mode site lists in my organization",
+      "requestUrl": "/beta/admin/edge/internetExplorerMode/siteLists",
+      "docLink": "https://docs.microsoft.com/graph/api/internetexplorermode-list-sitelists?view=graph-rest-beta",
```

Comment:
Documentation links have now changed to `https://learn.microsoft.com/`. Could we effect this for the `docLink` values?

Reply:
Absolutely!